src/test/java/net/snowflake/client/jdbc/ResultSetLatestIT.java

```diff
@@ -79,6 +81,32 @@ public void testMemoryClearingAfterInterrupt() throws Throwable {
     closeSQLObjects(resultSet, statement, connection);
   }
 
+  /**
+   * This test should only be run manually because it takes between 3 and 5 minutes to run. If it's
+   * successful, it will end between 3 and 5 minutes. This tests that when there is high statement
+   * concurrency, the downloader no longer hangs while fetching prefetch chunks.
+   */
+  @Ignore
```

Comment:
we should not ignore an important test. You can set max number of retries, or decrease the duration of each retry to let this test run faster. And then you should set a timeout for this test and kill it if it hangs.

Reply:
Also, do we have code coverage test to make sure the newly added code has been covered?

Reply:
I have added a function to manually reduce the amount of memory available so that the unit test can run in under a minute. This will let us be able to run it without ignoring it. 

Yes, the new code is covered by this unit test. I can see this by 1. Checking the Sonarqube report generated below, and 2.) Seeing from the log files generated that the code is going through the new logic